rxlifecycle-android-lifecycle/src/main/java/com/trellow/relifecycle2/android/lifecycle/RxLifecycleObserver.java

```diff
@@ -0,0 +1,55 @@
+package com.trellow.relifecycle2.android.lifecycle;
+
+import android.arch.lifecycle.Lifecycle;
+import android.arch.lifecycle.LifecycleObserver;
+import android.arch.lifecycle.LifecycleOwner;
+import android.arch.lifecycle.OnLifecycleEvent;
+import android.support.annotation.CheckResult;
+
+import com.trello.rxlifecycle2.LifecycleProvider;
+import com.trello.rxlifecycle2.LifecycleTransformer;
+import com.trello.rxlifecycle2.RxLifecycle;
+import com.trellow.relifecycle2.RxLifecycleAndroidLifecycle;
+
+import javax.annotation.Nonnull;
+
+import io.reactivex.Observable;
+import io.reactivex.subjects.BehaviorSubject;
+
+public class RxLifecycleObserver implements LifecycleProvider<Lifecycle.Event>, LifecycleObserver {
```

Comment:
Open to a better name. This wraps a `LifecycleOwner` and provides a `LifecycleProvider`.